packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx

```diff
@@ -157,14 +157,16 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps) {
             <div ref={ref} {...overlayProps} {...dialogProps}>
               <div className={styles.content} ref={refs.setFloating} style={floatingStyles} {...getFloatingProps()}>
                 <div className={styles.body}>
-                  <CustomScrollbar className={styles.leftSide} hideHorizontalTrack>
-                    <TimeRangeList
-                      title={t('time-picker.time-range.example-title', 'Example time ranges')}
-                      options={validOptions}
-                      onChange={onChangeTimeOption}
-                      value={timeOption}
-                    />
-                  </CustomScrollbar>
+                  <div className={styles.leftSide}>
+                    <ScrollContainer showScrollIndicators>
```

Comment:
Any reason we don't have `className` prop on `ScrollContainer?

Reply:
same reason we don't have it on any other of our components really. we don't want handcrafted css interfering with our component styles. it's still in `unstable` so if we find we actually need it we can add it

Reply:
I thought we've discussed somewhere that it's ok to expose `className`, but maybe I misremember 🤔